### PR TITLE
replaced phly/http with zendframework/zend-diactoros and updated tests.

### PR DESCRIPTION
**Pull request by websoftwares**

### CODE REVIEW

**Comment by websoftwares** [comment]
The code coverage was timed out on  Scrutinizer due to timeout from travis-ci u could restart the inspection on  Scrutinizer.

File: composer.json [hunk 1]
```diff
@@ -18,11 +18,11 @@
...
     "require": {
         "php": ">=5.5.0",
-        "psr/http-message": "~0.0",
+        "psr/http-message": "~1.0.0",
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "phly/http": "~0.0"
+        "zendframework/zend-diactoros": "1.0.0"
     },
```
Comment (harikt) [inline composer.json:26]: What about adding suggest as in PR #91

**Comment by harikt** [comment]
Thank you. Good PR :+1:  .

**Comment by pmjones** [comment]
Agh -- didn't see this PR until after I did it myself. Sorry about that. :-/

**Comment by websoftwares** [comment]
No problem, thanks for updating the dependencies, needed it for some projects :)

**Pull request closed without merging**